src/XIVLauncher.Common.Unix/UnixDalamudRunner.cs

```diff
@@ -74,43 +74,57 @@ public UnixDalamudRunner(CompatibilityTools compatibility, DirectoryInfo dotnetR
         launchArguments.Add(gameArgs);
 
         var dalamudProcess = compatibility.RunInPrefix(string.Join(" ", launchArguments), environment: environment, redirectOutput: true, writeLog: true);
-        var output = dalamudProcess.StandardOutput.ReadLine();
+        var output = dalamudProcess.StandardOutput.ReadLine() ?? "";
```

Comment:
`output` is checked against null later, no need to null-coalesce it twice.
Also `output` being null is meaningful information, since it indicates that the process exited without writing a single line to stdout